{
  "name": "study_async-and-catch-error",
  "version": "1.0.0",
  "main": "index.js",
  "type": "commonjs",
  "scripts": {
    "test": "jest --no-cache"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/BeigeWoods/Study_async-and-catch-error.git"
  },
  "keywords": [
    "async",
    "promise",
    "catch"
  ],
  "author": "BeigeWoods",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/BeigeWoods/Study_async-and-catch-error/issues"
  },
  "homepage": "https://github.com/BeigeWoods/Study_async-and-catch-error#readme",
  "description": "",
  "devDependencies": {
    "@types/jest": "^29.5.12",
    "jest": "^29.7.0"
  }
}
